Sources/App/Features/Slots/Models/SlotResponse.swift

```diff
@@ -10,7 +10,7 @@ import Vapor
 
 struct SlotResponse: Content {
     let id: UUID?
-    let startTime: Date?
+    let startTime: String?
```

Comment:
As this defaults to an empty string, can we make it mandatory rather than optional?

Reply:
Good spot!

In the follow up work to add the event I've made changes so that this is typed correctly as `String` - they're not optional in the DB so its safe 😄 

Thats merged in here now so you won't have to do two different client side changes, hopefully saves a little work!

Reply:
Nice one 👍